sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -94,6 +96,41 @@ FormRecognizerClient formRecognizerClient = new FormRecognizerClientBuilder()
 credential.update("{new_key}");
 ```
 
+#### Create a Form Recognizer client using Microsoft identity platform (formerly Azure Active Directory)
+
+Azure SDK for Java supports an Azure Identity package, making it easy to get credentials from Microsoft identity
+platform. First, add the package:
+
+[//]: # ({x-version-update-start;com.azure:azure-identity;current})
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-identity</artifactId>
+    <version>1.0.3</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+All the implemented ways to request a credential can be found under the `com.azure.identity.credential` package. The
```

Comment:
Is there a new guidance to include Identity section in the README's?

Reply:
+1